doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -486,7 +485,7 @@ can be useful in urderstanding the format.
       "type": "Point",
       "coordinates": [2.186447514, 41.3763726]
     },
-    "type": "application/vnd.geo+json"
+    "type": "geo:json"
```

Comment:
FYI @jmcanterafonseca 

I have changed this due to two reasons:

* The main one is that `/` is a forbidden char in attribute type (see https://github.com/telefonicaid/fiware-orion/blob/develop/doc/apiary/v2/fiware-ngsiv2-reference.apib#L309)
* In addition, it align the GeoJSON support with the other entity location shapes in the Simple Location Format, as all them start with `geo:`. Thus, using `geo:json` for GeoJSON seems to be a smart solution.